docs/public/_redirects

```diff
@@ -389,6 +389,8 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /base/api/trap-focus/ /base/api/focus-trap/ 301
 /material-ui/experimental-api/css-variables/ /material-ui/experimental-api/css-theme-variables/overview/ 301
 /joy-ui/main-features/perfect-dark-mode/ /joy-ui/main-features/dark-mode-optimization/ 301
+/pt/* /:splat 301
+/zh/* /:splat 301
```

Comment:
```suggestion
/pt/material-ui/* /material-ui/:splat 301
/zh/material-ui/* /material-ui/:splat 301

/pt/joy-ui/* /joy-ui/:splat 301
/zh/joy-ui/* /joy-ui/:splat 301

/pt/base/* /base/:splat 301
/zh/base/* /base/:splat 301

/pt/system/* /system/:splat 301
/zh/system/* /system/:splat 301
```

We can do this instead to avoid https://github.com/mui/material-ui/pull/34820/files#r999179500 and keep `/x` redirects independent.
This will allow us to disable translations in MUI X in our own pace.
What do you think?

Reply:
Makes sense 👍 Would the translation on x still work if this PR gets merged tough?

Reply:
BTW now that this is committed, make sure to add the redirects in https://github.com/mui/mui-x/pull/6560

Reply:
> Makes sense 👍 Would the translation on x still work if this PR gets merged tough?

I think so - we still have `LanguageNegotiation` in our docs.

Reply:
cc @bharatkashyap should we do something similar for toolpad? Is there support for translations there as well?

Reply:
No, @mnajdova, there's no need for this in Toolpad since we don't have translations at the moment.